lib/octokit/client/checks.rb

```diff
@@ -1,210 +1,151 @@
+      # @param check_run_id [Integer] The ID of the check run
+      # @return [Array<Sawyer::Resource>] A list of annotations
+      # @see https://developer.github.com/v3/checks/runs/#list-annotations-for-a-check-run
+      def check_annotations(repo, check_run_id, options = {})
+        opts = options
+        opts[:accept] = "application/vnd.github.antiope-preview+json" if opts[:accept].nil?
 
-      # Set preferences for check suites on a repository
-      #
-      # @param repo [Integer, String, Hash, Repository] A GitHub repository
-      # @param options [Hash] Preferences to set
-      # @return [Sawyer::Resource] A hash representing the repository's check suite preferences
-      # @see https://developer.github.com/v3/checks/suites/#set-preferences-for-check-suites-on-a-repository
-      # @example Set preferences for check suites on a repository
-      #   result = @client.set_check_suite_preferences("octocat/Hello-World", auto_trigger_checks: [{ app_id: 76765, setting: false }])
-      #   result.preferences.auto_trigger_checks.count # => 1
-      #   result.preferences.auto_trigger_checks[0].app_id # => 76765
-      #   result.preferences.auto_trigger_checks[0].setting # => false
-      #   result.repository.full_name # => "octocat/Hello-World"
-      def set_check_suite_preferences(repo, options = {})
-        ensure_api_media_type(:checks, options)
-
-        patch "#{Repository.path repo}/check-suites/preferences", options
+        paginate "#{Repository.path repo}/check-runs/#{check_run_id}/annotations", opts
       end
 
-      # Create a check suite
+      # Rerequest check suite
       #
-      # @param repo [Integer, String, Hash, Repository] A GitHub repository
-      # @param head_sha [String] The SHA of the commit to check
-      # @return [Sawyer::Resource] A hash representing the new check suite
-      # @see https://developer.github.com/v3/checks/suites/#create-a-check-suite
-      # @example Create a check suite
-      #   check_suite = @client.create_check_suite("octocat/Hello-World", "7638417db6d59f3c431d3e1f261cc637155684cd")
-      #   check_suite.head_sha # => "7638417db6d59f3c431d3e1f261cc637155684cd"
-      #   check_suite.status # => "queued"
-      def create_check_suite(repo, head_sha, options = {})
-        ensure_api_media_type(:checks, options)
-        options[:head_sha] = head_sha
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param check_suite_id [Integer] The ID of the check suite
+      # @return [Sawyer::Resource] The new suite
+      # @see https://developer.github.com/v3/checks/suites/#rerequest-check-suite
+      def rerequest_check_suite(repo, check_suite_id, options = {})
+        opts = options
+        opts[:accept] = "application/vnd.github.antiope-preview+json" if opts[:accept].nil?
 
-        post "#{Repository.path repo}/check-suites", options
+        post "#{Repository.path repo}/check-suites/#{check_suite_id}/rerequest", opts
```

Comment:
Should this be a boolean response?

Reply:
I was going to ask you this morning! This endpoint used to always return true, but there are similar endpoints we've done (such as https://developer.github.com/v3/projects/cards/#move-a-project-card) where we just return the response. Which way would we want to generate them?